app/steps/introduction_contact_information.rb

```diff
@@ -1,36 +1,27 @@
 # frozen_string_literal: true
 
 class IntroductionContactInformation < Step
-  step_attributes :phone_number,
-    :accepts_text_messages,
-    :email,
-    :mailing_street,
-    :mailing_city,
-    :mailing_zip,
-    :mailing_address_same_as_home_address
+  step_attributes(
+    :street_address,
+    :city,
+    :county,
+    :state,
+    :zip,
+  )
 
-  validates :email,
-    format: {
-      with: /\A.+@.+\..+\z/,
-      message: "Make sure you enter a valid email address",
-    },
-    allow_blank: true
-
-  validates :phone_number,
-    length: { is: 10, message: "Make sure your phone number is 10 digits long" }
+  validates :street_address,
+    presence: { message: "Make sure to provide a street address" }
 
-  validates :mailing_zip,
-    length: { is: 5, message: "Make sure your ZIP code is 5 digits long" },
-    allow_blank: true
+  validates :city,
+    presence: { message: "Make sure to provide a city" }
 
-  validates \
-    :accepts_text_messages,
-    :mailing_address_same_as_home_address,
-    presence: { message: "Make sure to answer this question" }
+  validates :county,
+    presence: { message: "Make sure to provide a county" }
 
-  before_validation :strip_non_digits_from_phone_number
+  validates :state,
+    presence: { message: "Make sure to provide a state" }
 
-  def strip_non_digits_from_phone_number
-    self.phone_number = phone_number.remove(/[^\d]/)
-  end
+  validates :zip,
+    length: { is: 5, message: "Make sure your ZIP code is 5 digits long" },
```

Comment:
We don't want to validate this is an integer, correct? Just wanted to make sure that was part of our permissive data plan.